pkgs/applications/networking/mailreaders/aerc/version.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/GNUmakefile b/GNUmakefile
+index 95556bc..05bb92d 100644
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -1,6 +1,6 @@
+ # variables that can be changed by users
+ #
+-VERSION ?= $(shell git describe --long --abbrev=12 --tags --dirty 2>/dev/null || echo 0.15.2)
++VERSION ?= $(shell git describe --long --abbrev=12 --tags --dirty 2>/dev/null || echo 0.16.0)
```

Comment:
What if we set this automatically from the derivation `version`? (For that matter, does it work to set `VERSION` directly in the derivation rather than using a patch?)